mcs/class/corlib/Mono/Runtime.cs

```diff
@@ -40,11 +40,26 @@ static class Runtime
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
 		private static extern void mono_runtime_install_handlers ();
 
-		static internal void InstallSignalHandlers ()
+#if MOBILE
+		public
+#else
+		internal
```

Comment:
Should this be #if MOBILE || XAMMAC_4_5? 

I'm assume we don't different behavior between XM Mobile and XM 4.5 target framework, and I know some desktop use cases care about error reporting frameworks.